RWrapper/mspc_1.0.1/DESCRIPTION

```diff
@@ -0,0 +1,19 @@
+Package: mspc
+Title: MSPC program
+Version: 1.0.0
```

Comment:
Version: 1.0.1

Reply:
I suggest we keep the version to 1.0.0 until it is release, and increment the version number at every release.

Reply:
you are right, then let's put 1.0.0 also in the folder name

Reply:
True, I guess we have two folders, `mspc_1.0.1` and `mspc_1.0.0`; maybe we need to merge them?!

Reply:
Oh sorry for the confusion. I used the name mspc_1.0.1 for my second version just to differentiate between the version of the package that contains the mspc software, and the one that does not, it's not really the name of the version of the package.

Reply:
Since we're not sure whether the package that has the MSPC executables will be accepted, I thought that we might wanna keep both versions. But we could merge them if you think it's better that way.

Reply:
I am guessing merging them and keeping only the one that will be submitted would be a better option.